### ISSUE TITLE
Visually hidden text for Brand - all services

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#2895]
**Is your feature request related to a problem? Please describe.**
The value of  `serviceLocalizedName` in service config e.g. https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/persian.js#L17 We should add other values to the relevant configs. 

**Describe the solution you'd like**
Update all service config files to include the `serviceLocalizedName` prop.
https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/
Pass that prop to the Brand component. See that the markup should be updated. 
e.g. 
```
<span role="text"><span lang="en-GB">BBC News</span>, Afrique</span>
```

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Checking with Storybook to see that the markup is correct https://bbc.github.io/simorgh/?path=/story/containers-brand--default
e.g.
```
<span role="text"><span lang="en-GB">BBC News</span>, Afrique</span>
```

**Additional context**
N/A

**Comment by sareh** [comment]
Closed as fixed.
